tests/parsing/GlobbingTest.py

```diff
@@ -195,8 +195,8 @@ def test_questionmark(self):
 
     def test_asterisk(self):
         pattern = 'a*b'
-        matches = ['axb', 'ayb']
-        non_matches = ['aXbX', os.path.join('a', 'b')]
+        matches = ['axb', 'ayb', os.path.join('a', 'b')]
+        non_matches = ['aXbX']
```

Comment:
Why do you have to change the position of ``os.path.join('a', 'b')`` from ``non_matches`` to ``matches``?

Reply:
the commit body explains that